bravado/client.py

```diff
@@ -215,6 +215,16 @@ def construct_params(self, request, op_kwargs):
             if not remaining_param.required and remaining_param.has_default():
                 marshal_param(remaining_param, None, request)
 
+    def log_warning_for_deprecated_op(self):
+        # TODO: Move this logic to bravado_decorators
+        if self.operation.op_spec.get('deprecated'):
+            params = {'op': self.operation.operation_id,
+                      'dep': self.operation.op_spec.get('x-deprecated-date'),
+                      'rem': self.operation.op_spec.get('x-removal-date'),
+                      }
+            log.warn(("[DEPRECATED] {op} has now been deprecated. Deprecation"
```

Comment:
As per [SO answer](http://stackoverflow.com/a/14762106) should this be changed to `warnings.warn`?